cmd/core-command/toml_test.go

```diff
@@ -1,25 +0,0 @@
-//
```

Comment:
Why are toml_tests being removed?  Seems like these are valuable in that they ensure the embedded configuration files are valid during the build pipeline.

Reply:
The tests as written leverage code that is marked as deprecated and exists just to run the tests. The value of ensuring that something checked in to source control exists and and is formatted correctly doesn't seem worth having all this extraneous code.

If there's a problem with the TOML, we can identify that in other ways. We don't need a scaffold for that.